### PR TITLE
Added changelog 1.1.2

### PR DESCRIPTION
**Pull request by RAD750**

### CODE REVIEW

File: _posts/2023-11-09-1.1.2-release.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+---
+layout: post
+title:  "Release 1.1.2"
+date:   2023-11-10 00:00:00 +0200
+tags: release
+author: lego11
+---
+
+Yet another update has been beamed to your computers now!
+This time starring:
```
Comment (Aang23) [inline _posts/2023-11-09-1.1.2-release.md:10]: typo

File: _posts/2023-11-09-1.1.2-release.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+# New features
+
+## Automation improvements
+
+It is now possible to very easily build an automated station for APT or LRPT satellites, advantageously replacing previous setups that required anywhere from 5 to 12 separate pieces of software.
+
+lego11 has made [a detailed configuration guide](https://www.a-centauri.com/articoli/automating-satellite-reception-with-satdump) for such a station.
```
Comment (Aang23) [inline _posts/2023-11-09-1.1.2-release.md:18]: do we move the guide on docs.satdump.org already or not?

File: _posts/2023-11-09-1.1.2-release.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+It is possible to have an alpha channel in the composite's equations, making it possible to build transparent composites that are designed to be overlaid on top of other channels.
+
+A typical use case would be overlaying MHS data over AVHRR, for example. 
+
+A `Rainfall (transparent)` composite that makes use of the alpha channel is already included in SatDump if you want to try it out.
+
+![Alt text](/assets/release_112/mhs.png)
+
+Projections are also supported with this new system.
```
Comment (Aang23) [inline _posts/2023-11-09-1.1.2-release.md:42]: (and are kinda the main point) - maybe add an example

File: _posts/2023-11-09-1.1.2-release.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+## New APT projections
+
+Aang23 remade the APT projections, which are now much better than with the previously used algorithm.
+
+## JPSS-1 DNB projections
+
+JPSS-1's DNB channel can now be projected correctly.
```
Comment (Aang23) [inline _posts/2023-11-09-1.1.2-release.md:50]: Another category to add : JPSS/HRD now have RS checks enabled by default as data integrity matters more than a bit more data due to the compression.

File: _posts/2023-11-09-1.1.2-release.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+
+## ATMS calibration
+
+ATMS is now calibrated on all JPSS satellites (including little SUOMI NPP) and can be used for science work, providing higher resolution and more channels than our other calibrated microwave imager, MHS.
```
Comment (Aang23) [inline _posts/2023-11-09-1.1.2-release.md:54]: maybe 1-2 example?

File: _posts/2023-11-09-1.1.2-release.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+
+## Arbitrarily long APT passes support
+
+SatDump now supports arbitrarily long APT passes, which could happen if you forgot to stop the recording before switching to another satellite. Previously, the limit was 15 minutes.
+
+The decoded image preview has been updated to also support this, so the image will not be cut off!
+
+## GUI backend improvements
+
+JVital2013 has made several improvements to the GUI system, improving compatibility with older graphic cards and Android systems and reducing the chance of the dreaded gray or corrupted screens appearing.
```
Comment (Aang23) [inline _posts/2023-11-09-1.1.2-release.md:64]: also greatly improved Windows-7 support

File: _posts/2023-11-09-1.1.2-release.md [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+
+If the HackRF was in use by another program, SatDump could crash. This was fixed.
+
+## Toast notification crash
+
+Too many toast notifications could cause a crash in certain circumstances. That has been fixed too.
+
+## Bug with Aqua
+
+A bug has been fixed with the Aqua satellite decoder.
```
Comment (Aang23) [inline _posts/2023-11-09-1.1.2-release.md:90]: *introduced in 1.1.1

File: _posts/2023-11-09-1.1.2-release.md [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+
+## Ephemeris fixes on JPSS
+
+Corrupted ephemeris data could lead to unexpected results on JPSS projections. This is no longer the case.
```
Comment (Aang23) [inline _posts/2023-11-09-1.1.2-release.md:94]: Goes back to the RS changes earlier + some additional checks

File: _posts/2023-11-09-1.1.2-release.md (hunk 2 above)
Comment (JVital2013) [reply _posts/2023-11-09-1.1.2-release.md:18]: I'd say yes

File: _posts/2023-11-09-1.1.2-release.md (hunk 6 above)
Comment (JVital2013) [reply _posts/2023-11-09-1.1.2-release.md:64]: I'll get screenshots soon

Comment (JVital2013) [reply _posts/2023-11-09-1.1.2-release.md:64]: OpenGL

**Comment by JVital2013** [comment]
Screenshot of despeckle

image

**Pull request merged**